components/analytics/team_analytics/team_analytics.tsx

```diff
@@ -323,13 +345,13 @@ export default class TeamAnalytics extends React.PureComponent {
     }
 }
 
-export function formatRecentUsersData(data, locale) {
+export function formatRecentUsersData(data: any, locale: any) {
```

Comment:
Looks like `data` in this case is `UserProfile[]` and `locale` is `string`.

Couple helpful hints:
- We should be avoiding using `any` in almost all circumstances, as that doesn't enforce any strict typing. `any` is okay in some of the test files.
- We have a library called `mattermost-redux` that contains the type definitions for most of the types we use in the application, so if you find yourself needing `any` check that library first.